Jenkinsfile

```diff
@@ -165,6 +168,10 @@ void importPackage(importPropertyFile, customProperties) {
   }
 }
 
+void setPipelineMetricsFlag(filePath) {
```

Comment:
Should we have a parameter for the actual value to be set as well? Like if we want to disable metrics and enable it for internal testing, it might be easier to have this function take in the actual boolean value as a parameter too and set it when we want to toggle the metrics during internal testing